packages/graph/hash_graph/tests/test_data/property_type/mod.rs

```diff
@@ -1,8 +1,10 @@
 pub const AGE_V1: &str = include_str!("age.json");
-pub const CONTACT_INFORMATION_V1: &str = include_str!("contact-information_v1.json");
-pub const CONTACT_INFORMATION_V2: &str = include_str!("contact-information_v2.json");
+pub const CONTACT_INFORMATION_V1: &str = include_str!("contact-information.json");
 pub const CONTRIVED_PROPERTY_V1: &str = include_str!("contrived-property.json");
 pub const FAVOURITE_QUOTE_V1: &str = include_str!("favourite-quote.json");
+pub const NAME_V1: &str = include_str!("name.json");
 pub const INTERESTS_V1: &str = include_str!("interests.json");
 pub const NUMBERS_V1: &str = include_str!("numbers.json");
-pub const USER_ID_V1: &str = include_str!("user-id.json");
+pub const TEXT_V1: &str = include_str!("text.json");
+pub const USER_ID_V1: &str = include_str!("user-id_v1.json");
```

Comment:
I get why we picked this naming but should we either swap these to all kebab-case or all snake_case? Because this is not bueno

Reply:
I tried to distinguish between the actual type and the version. I don't have a preference.

Reply:
The rest of our files and folders are snake_case so lets go with that

Reply:
Applied in 090b1971bbe1501dcf6ca503b33313dfa456569d